src/pages/api/ipfs/logo.page.ts

```diff
@@ -15,42 +15,31 @@ const handler = async (req: NextApiRequest, res: NextApiResponse) => {
   try {
     const incoming = new formidable.IncomingForm()
 
-    incoming.parse(req, async (err, fields, files) => {
-      if (err) {
-        return res.status(500).json({
-          error: t`Error parsing form`,
-        })
-      }
-      const file = files.file as formidable.File
-      const stream = fs.createReadStream(file.filepath)
-      let options
-      if (fields.pinataMetadata) {
-        const pinataMetadata = JSON.parse(fields.pinataMetadata as string)
-        options = {
-          pinataMetadata,
+    const result = await new Promise((resolve, reject) => {
```

Comment:
Re-ordered some of the code here as I don't think it was working as intended

Reply:
yep, you're correct 😂 was never actually returning anything cos the returns were in the `parse` callback